manifests/agent/postfix_ipv4.pp

```diff
@@ -1,4 +1,7 @@
 class classroom::agent::postfix_ipv4 {
+  package {'postfix':
```

Comment:
is this being installed so that the package doesn't need to be cached? Or should it be in a separate pr?